examples/foundational/32-gemini-grounding-metadata.py

```diff
@@ -33,7 +32,7 @@
 
 
 # Function handlers for the LLM
-search_tool = {"google_search_retrieval": {}}
+search_tool = {"google_search": {}}
```

Comment:
You will probably need to make the same changes inside the `news-chatbot` example. We are also using `google_search_retrieval` there.

Reply:
Thanks! Updated!